azerothcore/azerothcore-wotlk
# fix(DB/Quest): allows both factions to do Past Endeavors

## Changes Proposed:
-  Allows both factions to do Past Endeavors, as opposed to before where it was only Horde

## Issues Addressed:

- Closes #8784 
- Closes https://github.com/chromiecraft/chromiecraft/issues/2205

## SOURCE:



## Tests Performed:

- Done as described below
- Also tried to do quest after a faction change, couldn't do it

## How to Test the Changes:

Do the following steps for both sides and see it's available for both sides

1.    .q add 5056
2.    .q compete 5056
3.    .go c id 10303
4.    Turn in 5056 (Shy-Rotam)
5.    See if 5057 (Past Endeavors) is available


## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
